### PR TITLE
Document optional dependencies

### LINKED ISSUE #69: Document extras in installation docs
Based on comment from https://github.com/pyOpenSci/software-submission/issues/84

### PR DESCRIPTION
**Pull request by Midnighter**
* [x] fix #69 
* [x] description of feature/fix

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,24 @@ and thus automatically generated
 [Singularity](https://depot.galaxyproject.org/singularity/)
 [BioContainers](https://biocontainers.pro/) images also exist.
 
+### Optional Dependencies
+
+Taxpasta supports a number of extras that you can install for additional features. You
```
Comment (jfy133) [inline README.md:50]:
```suggestion
Taxpasta supports a number of extras that you can install for additional features - primarily support for additional output file formats. You
```

File: docs/index.md [hunk 2]
```diff
@@ -56,6 +56,24 @@ and thus automatically generated
 [Singularity](https://depot.galaxyproject.org/singularity/)
 [BioContainers](https://biocontainers.pro/) images also exist.
 
+### Optional Dependencies
+
+Taxpasta supports a number of extras that you can install for additional features. You
```
Comment (jfy133) [inline docs/index.md:61]:
```suggestion
Taxpasta supports a number of extras that you can install for additional features - primarily support for additional output file formats. You
```

**Review by jfy133 — APPROVED** [review]
One clarification suggestion

**Comment by jfy133** [comment]
Does the list of extras for development include the extra packages needed for rendering the website?

**Comment by Midnighter** [comment]
> Does the list of extras for development include the extra packages needed for rendering the website?

Good point, they don't. Those are separately in `docs/requirements.txt`. Let's see how we can do this. 🤔

**Comment by Midnighter** [comment]
I've added another extra `docs` for those dependencies. So at least it's within the same system but for now separate from the pure `dev` dependencies. I did this only because historically I've been keeping them apart. They could easily be combined, however.

**Pull request merged**